test/useFormo.test-d.ts

```diff
@@ -68,3 +74,50 @@ import { Option } from "fp-ts/Option";
 
   expectType<Record<"name" | "age", Option<NonEmptyArray<never>>>>(fieldErrors);
 }
+
+function wrapFormo<
+  Values extends Record<string, unknown>,
+  Validators extends Partial<FieldValidators<Values>>,
+  ArrayValidators extends Partial<FieldArrayValidators<Values>>,
+  FormErrors
+>(...args: FormOptions<Values, Validators, ArrayValidators, FormErrors>) {
+  const formo = useFormo(args[0], args[1]);
+
+  return formo;
+}
+
+{
+  const { fieldProps, fieldErrors } = wrapFormo(
+    {
+      initialValues: {
+        name: "",
+        age: 23,
+      },
+      fieldValidators: () => ({
+        name: validators.maxLength(10, false),
+        age: validators.validator((i: number) =>
+          i >= 18 ? either.right(i.toString()) : either.left(false)
+        ),
+      }),
+    },
+    {
+      onSubmit: (values) => {
+        expectType<string>(values.name);
+        expectType<string>(values.age);
+        return taskEither.left(values);
+      },
+    }
+  );
+
+  expectType<string>(fieldProps("name").value);
+  expectType<(v: string) => unknown>(fieldProps("name").onChange);
+  expectType<Option<NonEmptyArray<boolean>>>(fieldProps("name").issues);
```

Comment:
I tried with `expectNotType<Option<NonEmptyArray<never>>>(fieldProps("name").issues);` but it doesn't work and this check will not signal any error if issues is `never`